fern/api/definition/account.yml

```diff
@@ -0,0 +1,49 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/fern-api/fern/main/fern.schema.json
+
+imports:
```

Comment:
Could we move this to an `internal` folder inside `definitions` just like the `crm` folder? There would be more services here that would come in very soon.